src/m_menu.c

```diff
@@ -4197,6 +4192,10 @@ setup_menu_t gen_settings5[] = {
 
   {"", S_SKIP, m_null, M_X, M_SPC},
 
+  {"View Bobbing", S_THERMO|S_PCT, m_null, M_X_THRM8, M_SPC, {"view_bobbing_pct"}},
```

Comment:
Maybe we should move this to the "Weapons" menu? I'm not sure if that makes sense, but I feel like it should be next to "Weapon Bobbing". Alternatively, we could move them both to the General->QOL group.

Reply:
If I had to choose one of your two suggestions, it'd be the latter; general view bobbing is obviously not a "Weapons"-specific mechanic, so I don't think the setting belongs in that menu, whereas weapon bobbing _could_ be in the QOL group.

Furthermore, while having general view bobbing in the General menu and weapon bobbing in the Weapons menu sounds logical, I agree on that the settings should be grouped together.